core/include/tangram/map.h

```diff
@@ -65,20 +65,19 @@ struct SceneUpdate {
 };
 
 enum Error {
+    none,
     scene_update_path_not_found,
     scene_update_path_yaml_syntax_error,
     scene_update_value_yaml_syntax_error,
 };
 
-struct SceneUpdateError {
+struct SceneError {
```

Comment:
Maybe we should do that.. this `map.h` is getting a lot of stuff now. But maybe this work belongs in another refactor PR.